scripts/workflows/provision-darwin.sh

```diff
@@ -37,6 +37,8 @@ fi
 BATS_SUPPORT="/usr/local/lib/bats-support"
 echo "BATSLIB=${BATS_SUPPORT}" >> "$GITHUB_ENV"
 
-
+cargo build -p icx-asset
+ICX_ASSET="$(pwd)/target/debug/icx-asset"
+echo "ICX_ASSET=$ICX_ASSET" >> "$GITHUB_ENV"
```

Comment:
Let's do this only for the test that needs it:
```suggestion
if [ "$E2E_TEST" = "tests-icx-asset/icx-asset" ]; then
    cargo build -p icx-asset
    ICX_ASSET="$(pwd)/target/debug/icx-asset"
    echo "ICX_ASSET=$ICX_ASSET" >> "$GITHUB_ENV"
fi
```